Mathlib/Algebra/BigOperators/Finsupp.lean

```diff
@@ -551,6 +551,38 @@ theorem prod_embDomain [Zero M] [CommMonoid N] {v : α →₀ M} {f : α ↪ β}
 #align finsupp.prod_emb_domain Finsupp.prod_embDomain
 #align finsupp.sum_emb_domain Finsupp.sum_embDomain
 
+lemma exists_mem_embDomain_eq [Zero M] {f : α ↪ β} {g : β →₀ M} :
+    (∃ (v : α →₀ M), embDomain f v = g) ↔ (g.support : Set β) ⊆ (Set.range f) := by
```

Comment:
This doesn't belong in `BigOperators`; can you put it in `Finsupp.Basic` instead?

Reply:
OK. It is at a slightly bizarre place because, making use of your next comment, it has to be included after docs#Finsupp.comapDomain